### DISCUSSION TITLE
Phase 7 - publisher tag formalized

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#636] [Announcements - Changes Made]
The publisher tag is now formalized into the namespace:

https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#publisher

Discussion here:

https://github.com/Podcastindex-org/podcast-namespace/discussions/579

**Comment by samsethi** [comment]
Hi Dave 

Glad to see this formalized.  In the example for Publisher feeds it might be good idea to also show a fully formed Publisher Feed. i.e what's at the end of the Publisher Feed URL.  This example is a medium=music feed with additional support for the person tag and title attribute in the remote item.  Some people are asking for an img attribute also. 

I am happy to post here examples for other medium such as audiobooks, courses etc. if people want it. 

<img width="1434" alt="producers-65947e024b43e2b9db29ab03__2__xml" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/1448af39-3898-4d77-8216-dc617845a2c2">
.

**Reply by samsethi** [reply]
Here is a new medium=event publisher feed we are working on. 

<img width="1213" alt="New_Message" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/2d83b369-f6a4-4124-bef8-c46c3562ae32">
